proto/api.proto

```diff
@@ -45,6 +45,24 @@ message ClientCmdID {
   optional int64 random = 2 [(gogoproto.nullable) = false];
 }
 
+// ReadConsistencyType specifies what type of consistency is observed
+// during read operations.
+enum ReadConsistencyType {
+  option (gogoproto.goproto_enum_prefix) = false;
+  // CONSISTENT reads are guaranteed to read committed data; the
+  // mechanism relies on clocks to determine lease expirations.
+  CONSISTENT = 0;
+  // CONSENSUS requires that reads must achieve consensus. This is a
+  // stronger guarantee of consistency than CONSISTENT.
+  //
+  // TODO(spencer): current unimplemented.
+  CONSENSUS = 1;
```

Comment:
To pass Jepsen tests which mess with clocks to foil read leases. I'd like to just leave it in as a placeholder.